pkgs/top-level/python-packages.nix

```
@@ -10529,6 +10541,8 @@ self: super: with self; {
 
   pydantic-settings = callPackage ../development/python-modules/pydantic-settings { };
 
+  pydantic-yaml-0 = callPackage ../development/python-modules/pydantic-yaml-0 { };
```

We only allow multiple versions in `python-packages.nix` in very rare cases. Why was this done?
Anyway, it should have been
```suggestion
pydantic-yaml_0 = callPackage ../development/python-modules/pydantic-yaml/0.nix { };
```

---

I'm rectifying this in https://github.com/NixOS/nixpkgs/pull/329667.

---

Hey! Nice, thank you. I'll take a look!